test/Microsoft.TemplateEngine.IDE.IntegrationTests/TemplatePackagesTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.TemplateEngine.Abstractions.Installer;
+using Microsoft.TemplateEngine.Abstractions.TemplatesSources;
+using Microsoft.TemplateEngine.IDE.IntegrationTests.Utils;
+using Xunit;
+
+namespace Microsoft.TemplateEngine.IDE.IntegrationTests
+{
+    public class TemplatePackagesTests : IClassFixture<PackageManager>
+    {
+        private PackageManager _packageManager;
+        public TemplatePackagesTests(PackageManager packageManager)
+        {
+            _packageManager = packageManager;
+        }
+
+        [Fact]
+        internal async Task CanInstallLocalNuGetPackage()
+        {
+            Bootstrapper bootstrapper = BootstrapperFactory.GetBootstrapper();
+            string packageLocation = _packageManager.PackProjectTemplatesNuGetPackage("microsoft.dotnet.common.projecttemplates.5.0");
+
+            InstallRequest installRequest = new InstallRequest
+            {
+                Identifier = Path.GetFullPath(packageLocation)
+            };
+
+            IReadOnlyList<InstallResult> result = await bootstrapper.InstallAsync(new[] { installRequest }, InstallationScope.Global, CancellationToken.None).ConfigureAwait(false);
+
+            Assert.Equal(1, result.Count);
+            Assert.True(result.First().Success);
+            Assert.Equal(InstallerErrorCode.Success, result.First().Error);
+            result.First().ErrorMessage.Should().BeNullOrEmpty();
+            Assert.Equal(installRequest, result.First().InstallRequest);
+
+            IManagedTemplatesSource source = result.First().Source;
+            Assert.Equal("Microsoft.DotNet.Common.ProjectTemplates.5.0", source.Identifier);
+            Assert.Equal("Global Settings", source.Provider.Factory.Name);
+            Assert.Equal("NuGet", source.Installer.Name);
+            Assert.Equal("Microsoft", source.GetDisplayDetails()["Author"]);
```

Comment:
@DavidKarlas it appears that `Details` collection is missing in `IManagedTemplatesSource`, only in `ITemplatesSource? Should we bring it back? 
Details are not accessible publicly now as implementations are internal, only details that can be retrieved by `GetDisplayDetails()`

Reply:
It feels like `GetDisplayDetails` are enough, no? What is benefit of having `Details` exposed?